### PR TITLE
replaced sscanf by atoi

### PR DESCRIPTION
**Pull request by OlegHahm**

### CODE REVIEW

**Comment by mehlis** [comment]
perhaps we should go for `strtol`, because:

> The atoi() function is subsumed by strtol() but is retained because it is used extensively in existing code. If the number is not known to be in range, strtol() should be used because atoi() is not required to perform any error checking.

from http://pubs.opengroup.org/onlinepubs/000095399/functions/atoi.html

**Comment by stze** [comment]
christian is right, atoi and related functions lack mechanisms for
reporting errors for invalid values. according to [1]
they:
- do not need to set errno on an error.
- have undefined behavior if the value of the result cannot be
  represented.
- return 0 if the string does not represent an integer, which is
  indistinguishable from a correctly formatted, zero-denoting input
  string.

[1] https://www.securecoding.cert.org/confluence/display/seccode/INT06-C.+Use+strtol()+or+a+related+function+to+convert+a+string+token+to+an+integer

**Comment by OlegHahm** [comment]
agreed and changed.

File: sixlowpan/main.c [hunk 1]
```diff
@@ -30,10 +30,11 @@ void init(char *str){
     char command;
     uint16_t r_addr;
     ipv6_addr_t std_addr;
+
+    command = str[5];
+    r_addr = strtol(str + 7, NULL, 10);
```
Comment (mehlis) [inline sixlowpan/main.c]: magic `7`? if you want the second argument, just call `strtok` twice...same for the magic `5` above

**Comment by OlegHahm** [comment]
Changed to strtok and adapted coding conventions

**Comment by OlegHahm** [comment]
Everyone fine with that?

**Comment by mehlis** [comment]
ACK

**Pull request merged**